roles/README.md

```diff
@@ -21,9 +21,9 @@ You may have also seen that we have people in Tech Lead and Tech Architect roles
 
 ### Delivery
 
-- [Delivery Manager](delivery_manager.md)
-- Senior Delivery Manager
-- [Delivery Principal](delivery_principal.md)
+- [Delivery Manager](delivery_manager.md) (SFIA 4)
+- Senior Delivery Manager (SFIA 5)
+- [Delivery Principal](delivery_principal.md) (SFIA 6)
```

Comment:
```suggestion
- [Team Assistant] (SFIA 2)
- [Junior Delivery Manager] (SFIA 3)
- [Delivery Manager](delivery_manager.md) (SFIA 4)
- Senior Delivery Manager (SFIA 5)
- [Delivery Principal](delivery_principal.md) (SFIA 6)
```

We do have some SFIA guidance for these delivery roles here: https://github.com/madetech/handbook/tree/main/roles/sfia

I'm not sure how outdated these are, but we could perhaps link them here in the meantime for general guidance.